src/code/z_actor.c

```diff
@@ -419,101 +403,95 @@ void func_8002C124(TargetContext* targetCtx, GlobalContext* globalCtx) {
     CLOSE_DISPS(globalCtx->state.gfxCtx, "../z_actor.c", 2158);
 }
 
-void func_8002C7BC(TargetContext* targetCtx, Player* player, Actor* actorArg, GlobalContext* globalCtx) {
+void Target_Update(TargetContext* targetCtx, Player* player, Actor* actorArg, GlobalContext* globalCtx) {
     s32 pad;
-    Actor* unkActor;
+    Actor* naviTarget = NULL; // Actor that Navi is currently targetting
```

Comment:
```suggestion
    Actor* targetedActor = NULL;
```

Reply:
This is actually specifically for the actor that Navi is focusing on, but not nessescarily what is being targeted (lock on)

Reply:
Then the word `target` shouldn't be used if it isn't necessarily related to the target system.

Reply:
how is it not related to the target system? 🤔 Navi is very intertwined with it, but anyways whenever Navi hovers over something the arrow also appears above it, but that does not mean it's locked into (targeted)